typing/mode.ml

```diff
@@ -2168,6 +2174,19 @@ module Const = struct
     let areality = C.locality_as_regionality areality in
     { areality; linearity; portability; uniqueness; contention }
 
+  module Axis = struct
+    type 'd packed_value_axis =
+      | P : ('m, 'a, 'd) Value.axis -> 'd packed_value_axis
```

Comment:
Maybe move this to `module Value_with`, right after `type axis`? Say `type axis_packed`?
Then the function below can convert `Alloc.axis_packed` to `Value.axis_packed` which is more symmetric.

Reply:
nice :)